### PR TITLE
implement login feature

### PR DESCRIPTION
**Pull request by mfasl2k**
1. Code Quality

- [x] Is the code easy to read and understand?

- [x] Are functions/methods short, clear, and focused on a single task?

- [ ] Is the code modular and reusable where appropriate?

2. Functionality

- [x] Does the code meet all the functional requirements?

- [x] Are all edge cases and error conditions handled?

- [x] Does it integrate correctly with existing code?

3. Style & Conventions

- [x] Does the code follow the team’s style guide (e.g., PEP8 for Python, Airbnb for JavaScript)?

- [x] Are naming conventions consistent and meaningful?

- [x] Are indentation and spacing consistent?

4. Testing

- [x] Are there unit and/or integration tests for new features?

- [x] Do the tests cover both normal and edge cases?

- [x] Do all tests pass?

5. Security

- [x] Are all user inputs properly validated and sanitized?

- [x] Are there any hardcoded credentials or secrets?

- [x] Is sensitive data handled securely?

6. Performance

- [x] Is the code efficient and optimized where needed?

- [x] Are there any unnecessary computations or database queries?

- [x] Are large datasets handled properly?

7. Documentation

- [x] Are code comments used where necessary to explain complex logic?

- [x] Are public methods and APIs documented?

- [x] Is the README or other project documentation updated (if needed)?

### CODE REVIEW

File: src/components/Register.js [hunk 1]
```diff
@@ -1,48 +1,103 @@
...
+            />
             <button type="submit">Register</button>
           </form>
         </div>
+        <div className="login">
+          <form onSubmit={handleLogin}>
+            <label htmlFor="chk" aria-hidden="true">
+              Login
+            </label>
+            <input type="email" name="email" placeholder="Email" />
```
Comment (light142) [inline src/components/Register.js:93]: this field should be required.
```suggestion
            <input type="email" name="email" placeholder="Email" required />
```

**Review by light142 — CHANGES_REQUESTED** [review]
please fix.

File: src/components/Register.js (hunk 1 above)
Comment (mfasl2k) [reply src/components/Register.js]: fixed ty!

**Review by light142 — APPROVED** [review]
looks all good
